libraries/protocol/include/steemit/protocol/steem_virtual_operations.hpp

```diff
@@ -166,6 +166,16 @@ namespace steemit { namespace protocol {
       asset             reward;
    };
 
+   struct producer_reward_operation : public virtual_operation
+   {
+      producer_reward_operation(){}
+      producer_reward_operation( const string& o, const asset& v ) : owner( o ), vesting_shares( v ) {}
+
+      account_name_type owner;
```

Comment:
Change the variable name to `witness` or `producer`. In the context of the witness object `owner` makes sense because the account name is the owner of the witness object. In the virtual op, it makes less sense.